inst/include/mode/solver.hpp

```diff
@@ -99,12 +98,25 @@ class solver {
     }
   }
 
+  void set_time(double t) {
+    reset(t);
```

Comment:
assuming that setting time means resetting everything; is this correct?

Reply:
Yes, I think that's a reasonable assumption - I can't think of a case where that would not hold.